tools/docker/bin/run.sh

```diff
@@ -80,6 +80,16 @@ if [ ! -f /tmp/wordpress-develop/wp-tests-config.php ]; then
 	cp /tmp/wp-tests-config.php /tmp/wordpress-develop/wp-tests-config.php
 fi
 
+# Symlink jetpack into the wp-content dir.
+if [ ! -e /var/www/html/wp-content/plugins/jetpack ]; then
```

Comment:
This doesn't work. tools/docker/docker-compose.yml has a hack to avoid recursion (look for "dockerdirectory") which creates this path, preventing the symlink.

Reply:
Fixed in 64f35ee